app/components/newsletter/newsletter-header-overview.js

```diff
@@ -181,6 +190,40 @@ export default class NewsletterHeaderOverviewComponent extends Component {
     this.showConfirmPublishAll = false;
   }
 
+  async canSendMailCampaign() {
+    let themisPublicationActivities = await this.args.meeting.themisPublicationActivities;
+    let themisPublicationActivity = themisPublicationActivities.find((activity) => activity.scope.includes('documents'));
+
+    let hasDocumentPublicationPlanned = isPresent(themisPublicationActivity?.plannedDate);
+    let hasNotas = false;
+    let hasThemes = false;
+
+    const agendaitems = await this.store.query('agendaitem', {
+      filter: {
+        agenda: {
+          ':id:': this.args.agenda.id,
+        },
+        'type': {
+          ':uri:': CONSTANTS.AGENDA_ITEM_TYPES.NOTA,
+        },
+      },
+      'page[size]': PAGE_SIZE.AGENDAITEMS,
+    });
+    for (const agendaitem of agendaitems.toArray()) {
+      hasNotas = true;
+
+      const agendaitemTreatment = await agendaitem.treatment;
+      const newsletterItem = await agendaitemTreatment.newsletterInfo;
+      const themes = await newsletterItem?.themes;
+      if (themes?.length) {
+        hasThemes = true;
+        break;
+      }
+    }
```

Comment:
Execute a `this.store.count`-query on `newsletter-info` with a filter on agenda-id and `:has:themes`? Or am I missing something?